src/Compiler/Driver/ParseAndCheckInputs.fs

```diff
@@ -426,6 +430,17 @@ let ParseInput
 
     try
         let input =
+            let identStore = HashSet<string>()
+
+            let identCaptureLexer x =
```

Comment:
Probably not the right way to do this, but so far the only thing I could come up with and quickly try...

Reply:
I believe FSharpToken has an IsIdentifier property, would that work?

Reply:
Perhaps I meant to ask why these:

                | Parser.token.PERCENT_OP ident
                | Parser.token.FUNKY_OPERATOR_NAME ident
                | Parser.token.ADJACENT_PREFIX_OP ident
                | Parser.token.PLUS_MINUS_OP ident
                | Parser.token.INFIX_AMP_OP ident
                | Parser.token.INFIX_STAR_DIV_MOD_OP ident
                | Parser.token.PREFIX_OP ident
                | Parser.token.INFIX_BAR_OP ident
                | Parser.token.INFIX_AT_HAT_OP ident
                | Parser.token.INFIX_COMPARE_OP ident
                | Parser.token.INFIX_STAR_STAR_OP ident

Reply:
We need those otherwise it won't find any operators.